jobs/openvpn/templates/etc/openvpn.conf.erb

```diff
@@ -26,6 +26,14 @@ server <%= p('openvpn.server') %>
 <% p('openvpn.push_routes').each do | route | %>
 push "route <%= route %>"
 <% end %>
+<% p('openvpn.push_dns').each do |dns| %>
+# ensure we can route to the DNS server over the connection
+push "route <%= dns %> 255.255.255.255"
```

Comment:
I can't think of a practical scenario where you wouldn't want to force the DNS traffic through the VPN, so I guess this is good convention to enforce.